workers/unity/Packages/com.improbable.gdk.tools/GenerateFiles.cs

```diff
@@ -0,0 +1,38 @@
+using System.IO;
+using System.Text.RegularExpressions;
+using UnityEditor;
+using UnityEngine;
+
+namespace Improbable.Gdk.Tools
+{
+    public class GenerateFiles
+    {
+        private static readonly string
+            ResourcesDir = Path.GetFullPath(Path.Combine(Application.dataPath, "Resources"));
+
+        [MenuItem("SpatialOS/Generate Dev Authentication Token", false, MenuPriorities.GenerateDevAuthToken)]
+        private static void GenerateDevAuthToken()
+        {
+            var receivedMessage = string.Empty;
+            RedirectedProcess
+                .Command(Common.SpatialBinary)
+                .WithArgs("project", "auth", "dev-auth-token", "create", "--description", "Dev Auth Token")
+                .InDirectory(Common.SpatialProjectRootDir)
+                .AddOutputProcessing((message) => receivedMessage += message)
+                .Run();
+            var regexString = @"""id\\"": \\""(.*?)\\""";
+            var regex = new Regex(regexString);
```

Comment:
Should we merge two above lines?

Reply:
That's worrying, and we shouldn't be working around invalid json.
However, when I run the command I see
```
{
        "id": "d237c0ed-b409-41f1-90f8-e185b8c3d0fd",
        "projectName": "unity_gdk",
        "description": "What",
        "creationTime": "2019-02-05T09:32:55.510358144Z",
        "expirationTime": "2019-03-07T09:32:55.510358144Z"
}
```
Which is completely valid json. What does Unity's JSONUtil do with the input json?